.gitignore

```diff
@@ -1,7 +1,10 @@
-# directory created by notebooks to store results
+# Directories created by notebooks to store data, results, etc.
+# "data" dir to store any data that is downloaded or otherwise needs to be made available to the notebook
+# occasionally, data may need to be checked into the repo (e.g., forced_photometry/data), but this should be rare
+data/
```

Comment:
This will not affect the files in forced_photometry/data that are already checked into the repo, but it will prevent any new files in a "data" directory from being accidentally committed.